pkg/yurtctl/cmd/join/join.go

```diff
@@ -159,21 +161,26 @@ func addJoinConfigFlags(flagSet *flag.FlagSet, joinOptions *joinOptions) {
 		&joinOptions.nodeLabels, options.NodeLabels, joinOptions.nodeLabels,
 		"Sets the labels for joining node",
 	)
+	flagSet.StringVar(
+		&joinOptions.kubernetesResourceServer, options.KubernetesResourceServer, joinOptions.kubernetesResourceServer,
+		"Sets the address for download k8s node resources",
```

Comment:
Maybe we should modify it as `Sets the address for downloading k8s node resources`

Reply:
Just correct it.